subprojects/core/src/main/java/org/gradle/execution/taskgraph/DefaultTaskExecutionPlan.java

```diff
@@ -613,6 +635,24 @@ private TaskInfo selectNextTask(final WorkerLease workerLease) {
         return selected.get();
     }
 
+    private TaskMutationInfo getUpdatedTaskMutationInfo(TaskInfo taskInfo) {
```

Comment:
Should this be `getResolvedTaskMutationInfo()`?

Reply:
renamed.